### PR TITLE
chore: add perf to readme

### PR DESCRIPTION
**Pull request by p-shahi**

### CODE REVIEW

**Review by marten-seemann — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,6 +19,10 @@ More specs to come soon!
 
 These test-plans historically used Testground. To read why we're now using `docker compose` instead please see: [Why we're moving away from Testground](https://github.com/libp2p/test-plans/issues/103)
 
+## Performance Benchmarking
+
+Please see the [libp2p perf protocol](https://github.com/libp2p/specs/blob/master/perf/perf.md) and the [benchmarking README](https://github.com/libp2p/test-plans/tree/master/perf#libp2p-performance-benchmarking).
```
Comment (thomaseizinger) [inline README.md:24]: Wouldn't this be better relative links?

```suggestion
Please see the [libp2p perf protocol](https://github.com/libp2p/specs/blob/master/perf/perf.md) and the [benchmarking README](./perf#libp2p-performance-benchmarking).
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-This repository contains interoperability and end to end tests for libp2p modules across different implementations and versions.
+This repository contains:
+* interoperability and end to end tests for libp2p modules across different implementations and versions
+* components to run performance benchmarks for different libp2p implementations
 
-## Specs
+## Multidimensional Interop
+### Specs
 
 Please see our first specification for interoperability tests between transports, multiplexers, and secure channels here: [Interoperability Test Specs](https://github.com/libp2p/test-plans/blob/master/multidim-interop/README.md)
```
Comment (thomaseizinger) [inline README.md:14]: This one too.

```suggestion
Please see our first specification for interoperability tests between transports, multiplexers, and secure channels here: [Interoperability Test Specs](./multidim-interop/README.md)
```

File: README.md (hunk 1 above)
Comment (mxinden) [inline README.md:24]: Note that the `perf/README.md` already points to the libp2p perf protocol specification. Slight preference for not linking to the specs repository twice.

**Review by mxinden — APPROVED** [review]
Thank you @p-shahi. I am in favor of Thomas suggestions.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
 
 More specs to come soon!
 
 ## History
 
 These test-plans historically used Testground. To read why we're now using `docker compose` instead please see: [Why we're moving away from Testground](https://github.com/libp2p/test-plans/issues/103)
 
+## Performance Benchmarking
+
+Please see the [libp2p perf protocol](https://github.com/libp2p/specs/blob/master/perf/perf.md) and the [benchmarking README](./perf#libp2p-performance-benchmarking).
```
Comment (p-shahi) [inline README.md:24]:
```suggestion
Please see the [benchmarking README](./perf#libp2p-performance-benchmarking).
```

**Pull request merged**